app/lib/metric_util.rb

```diff
@@ -0,0 +1,52 @@
+require "uri"
+require "net/http"
+
+# MetricUtil is currently used for posting metrics to Datadog's metrics endpoints.
+class MetricUtil
+  def self.put_metric_now(name, value)
+    put_metric(name, value, Time.now.to_i)
+  end
+
+  def self.put_metric(name, value, time)
+    # Time = POSIX time with just seconds
+    points = [[time, value]]
+    put_metric_point_series(name, points)
+  end
+
+  def self.put_metric_point_series(name, points)
+    name = "idseq.web.#{Rails.env}.#{name}"
+    data = JSON.dump("series" => [{
+                       "metric" => name,
+                       "points" => points
+                     }])
+    post_to_datadog(data)
```

Comment:
We could make it post to google analytics too... easier integration with the dashboard

Reply:
good idea, I will add the authorization/Google integration later